astropy/io/votable/converters.py

```diff
@@ -189,7 +189,8 @@ def parse(self, value, config=None, pos=None):
 
         Returns
         -------
-        native : tuple (value, mask)
+        native : (2,) tuple
```

Comment:
This draws from the numpy standard (https://numpy.org/doc/stable/glossary.html), so (n,) specifies the shape. I've added the numpy glossary term to ours.

Reply:
But according to the glossary it _denotes a tuple with **one** element_ – whereas the return here is a 2-tuple, and having the elements of the tuple listed seems more informative. Maybe I'd have to see an example in the numpy docs to understand the style better.

Reply:
I've reconfigured the explanation to avoid the shape argument. I do think it is worth revisiting, because it would be nice to be able to say 2 tuple or (3,3) array to succinctly convey the shape.